package.json

```diff
@@ -18,6 +18,7 @@
     "@astrojs/rss": "^4.0.7",
     "@astrojs/sitemap": "^3.1.6",
     "@astrojs/tailwind": "^5.1.0",
+    "@fastly/js-compute": "0.5.8",
```

Comment:
I think this is problematic in my local dev environment:

```sh
yarn && yarn run dev
yarn install v1.22.22
warning package.json: No license field
warning devolio@0.0.1: No license field
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
error @fastly/js-compute@0.5.8: The engine "node" is incompatible with this module. Expected version "^16 || ^18". Got "20.10.0"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

Reply:
[I looked at one of my prior setups from 7 months ago](https://github.com/sharunkumar/arun-portfolio/blob/16e9adeaa5026a82c650d1bebdea9560d3c034e9/package.json), and this dependency wasn't part of it back then but Fastly's Compute processes were part of the Github workflows.

Reply:
> I think this is problematic in my local dev environment:
> 
> ```shell
> yarn && yarn run dev
> yarn install v1.22.22
> warning package.json: No license field
> warning devolio@0.0.1: No license field
> [1/4] 🔍  Resolving packages...
> [2/4] 🚚  Fetching packages...
> error @fastly/js-compute@0.5.8: The engine "node" is incompatible with this module. Expected version "^16 || ^18". Got "20.10.0"
> error Found incompatible module.
> info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
> ```

your local node version is incompatible. use something like https://github.com/Schniz/fnm to manage node versions

Reply:
after installing and setting up fnm, run `fnm use 18`